src/github.com/getlantern/lantern-android/client/client.go

```diff
@@ -55,12 +53,14 @@ func NewClient(addr, appName string) *MobileClient {
 
 	hqfd := client.Configure(clientConfig.Client)
```

Comment:
The ``init()`` function is always implicitly called in Go, right? It's for general package initialization.